engine/src/main/java/org/terasology/rendering/dag/tasks/SetCullFaceModeTask.java

```diff
@@ -17,15 +17,15 @@
 
 import org.lwjgl.opengl.GL11;
 import org.terasology.rendering.dag.RenderPipelineTask;
-import org.terasology.rendering.dag.stateChanges.SetCullFace;
+import org.terasology.rendering.dag.stateChanges.SetCullFaceMode;
 
 /**
  * TODO: Add javadocs
  */
-public class SetCullFaceTask implements RenderPipelineTask {
+public final class SetCullFaceModeTask implements RenderPipelineTask {
```

Comment:
In line with a comment above I recommend to rename this to `SetWhatFacesToCullTask`.